staging/src/k8s.io/kubelet/pkg/apis/well_known_labels.go

```diff
@@ -42,9 +42,6 @@ var kubeletLabels = sets.NewString(
 	v1.LabelInstanceTypeStable,
 	v1.LabelOSStable,
 	v1.LabelArchStable,
-
-	LabelOS,
-	LabelArch,
```

Comment:
drop this change, these labels should still be considered valid for the kubelet to set in terms of authorization

Reply:
Updated.